Dockerfile.base

```diff
@@ -0,0 +1,38 @@
+FROM python:3.6-alpine
+LABEL maintainer "EGA System Developers"
+
+RUN apk add --no-cache \
+    libressl libressl-dev make gcc musl-dev libffi-dev postgresql-libs postgresql-dev git
+
+#################################################
+##
+## Install LocalEGA stuff
+##
+#################################################
+
+ARG LEGA_GID=1000
+RUN addgroup -g ${LEGA_GID} lega && \
+    adduser -D -G lega lega
+
+# RUN git clone https://github.com/EGA-archive/crypt4gh.git /root/crypt4gh && \
```

Comment:
Why is still around? If it isn't used it should be removed.

Reply:
Why is the commented out section still in the docker file?
If it not needed or used, remove it.

Reply:
It'll be used very soon.
It'll replace the deprecated localega-cryptor.